app/konnect/getting-started/productize-service.md

```diff
@@ -39,9 +39,9 @@ After creating the new version, you will see **Link with a Gateway Service** as
 
 1. Select **Link with a Gateway Service**. 
 
-    Choose a [runtime group](/konnect/runtime-manager/runtime-groups/) and [Gateway Service](/konnect/runtime-manager/configuration/#gateway-services) to
+    Choose a [control plane](/konnect/gateway-manager/control-plane-groups/) and [Gateway Service](/konnect/gateway-manager/configuration/#gateway-services) to
     deploy this API product version to. This lets you deploy to a specific group of
-    runtime instances in a specific environment.
+    data planes in a specific environment.
```

Comment:
It wont let me leave a suggestion here, but we should remove the use of specific twice here. Maybe "this lets you deploy your service across a data planes associated with the control plane"